README.md

```diff
@@ -9,6 +9,12 @@ mkdocs, with the following command (replace yum by dnf on Fedora):
 
     # yum install mkdocs
 
+Install mkdocs in Fedora 23+
+    
+    #pip install --upgrade pip
```

Comment:
It's better off if you to install pip first using `dnf`, then upgrade pip using pip, and later install `mkdocs`.

```sh
# dnf install python-pip
# pip install -U pip
# pip install mkdocs
```